db/migrate/20220114230942_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
+      t.string :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
```

Comment:
- [ ] We are missing the references to user table from post, every post will need to have its author, and that author is from table users, that's why you need to have a foreign key which is linking posts and users